integrationTest/servicePytest/pushServicePytest.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime, timedelta
+import pytest
+import requests
+import math
+import json
+import psycopg2
+import logging
+import os
+from time import sleep
+
+LOGGER = logging.getLogger(__name__)
+
+pytest.SERVER = "https://rocket-dev01.appspot.com"
+pytest.FAKE_STMO_ENDPOINT = "/test/uids?times=1"
+pytest.EQUEUE_PUSH_REQUEST_ENDPOINT = "/api/v1/admin/push/enqueue"
+
+
+@pytest.fixture
+def clean_db_entry(request):
+    DB_PWD = request.config.getoption("--DB_PWD")
+    conn = connect_database(DB_PWD)
+    conn_info = []
+    conn_info.append(conn)
+    yield conn_info
+    LOGGER.info(
+        'Clear up database entry where mozMessageId [{}]'.format(
+            conn_info[1]))
+    doDelete(conn, conn_info[1])
+    conn.close()
+
+
+def connect_database(DB_PWD):
+    """Connect to local database through proxy
+    """
+    hostname = '127.0.0.1'
+    username = 'pubsub'
```

Comment:
1. we can secure db name and user name to make it more secure.
2. Yes, the cloud proxy service builds up a secure proxy connection to cloud database so that the cloud build can connect this local host to target remote.